### PR TITLE
Plz review : @etix etix 

### PR DESCRIPTION
**Pull request by I-Cat**
+ copyright up date +2020
+ added etix avatar  yey

Plz review

@etix etix

### CODE REVIEW

File: cli/commands.go [hunk 1]
```diff
@@ -38,7 +38,7 @@ import (
 
 const (
 	commentSeparator  = "##### Comments go below this line #####"
-	defaultRPCTimeout = time.Second * 10
+	defaultRPCTimeout = time.Second * 20
```
Comment (etix) [inline cli/commands.go:41]: Why changing the RPC timeout?

File: contrib/init/systemd/.gitignore [hunk 2]
```diff
@@ -1 +1 @@
-mirrorbits.service
+mirrorbits.service
```
Comment (etix) [inline contrib/init/systemd/.gitignore:1]: No need to touch whitespaces.

File: mirrors/mirrors.go [hunk 3]
```diff
@@ -1,87 +1,67 @@
...
 import (
+	"bytes"
 	"fmt"
-	"math/rand"
 	"strconv"
 	"strings"
-	"time"
-
-	. "github.com/etix/mirrorbits/config"
+        "github.com/etix/mirrorbits/config"
```
Comment (etix) [inline mirrors/mirrors.go:12]: No, this breaks the build.

File: mirrors/mirrors.go [hunk 4]
```diff
@@ -1,87 +1,67 @@
...
 	"github.com/etix/mirrorbits/core"
 	"github.com/etix/mirrorbits/database"
 	"github.com/etix/mirrorbits/filesystem"
 	"github.com/etix/mirrorbits/network"
 	"github.com/etix/mirrorbits/utils"
 	"github.com/gomodule/redigo/redis"
+        "math/rand"
+        "time"
+
```
Comment (etix) [inline mirrors/mirrors.go:21]: Don't touch whitespaces please.

File: mirrors/mirrors.go [hunk 5]
```diff
@@ -1,87 +1,67 @@
...
-	CountryFields               []string         `redis:"-" json:"-" yaml:"-"`
-	ExcludedCountryFields       []string         `redis:"-" json:"-" yaml:"-"`
-	Filepath                    string           `redis:"-" json:"-" yaml:"-"`
-	Weight                      float32          `redis:"-" json:"-" yaml:"-"`
-	ComputedScore               int              `redis:"-" yaml:"-"`
-	LastSync                    Time             `redis:"lastSync" yaml:"-"`
-	LastSuccessfulSync          Time             `redis:"lastSuccessfulSync" yaml:"-"`
-	LastSuccessfulSyncProtocol  core.ScannerType `redis:"lastSuccessfulSyncProtocol" yaml:"-"`
-	LastSuccessfulSyncPrecision core.Precision   `redis:"lastSuccessfulSyncPrecision" yaml:"-"`
-	LastModTime                 Time             `redis:"lastModTime" yaml:"-"`
```
Comment (etix) [inline mirrors/mirrors.go:62]: Why did you remove variables? Doesn't make sense.

File: mirrors/mirrors.go [hunk 6]
```diff
@@ -1,87 +1,67 @@
...
-func (m *Mirror) Prepare() {
-	m.CountryFields = strings.Fields(m.CountryCodes)
-	m.ExcludedCountryFields = strings.Fields(m.ExcludedCountryCodes)
-}
-
-// IsHTTPS returns true if the mirror has an HTTPS address
-func (m *Mirror) IsHTTPS() bool {
-	return strings.HasPrefix(m.HttpURL, "https://")
-}
-
```
Comment (etix) [inline mirrors/mirrors.go:77]: Why?

File: README.md [hunk 7]
```diff
@@ -1,3 +1,5 @@
+
+[![@etix](https://avatars0.githubusercontent.com/u/38853?s=120&v=4)
```
Comment (etix) [inline README.md:2]: I'm not interested in having my avatar on the README.

**Review by etix — COMMENTED** [review]
Many of these changes are removing useful lines and variables, breaking the build, changing whitespaces, and all of that in 55 commits (!!).

I don't get the point of all these changes.

File: cli/commands.go (hunk 1 above)
Comment (I-Cat) [reply cli/commands.go:41]: The time out should be 20 as normal latency is AVG 20

File: README.md (hunk 7 above)
Comment (I-Cat) [reply README.md:2]: Ok thought it be very nice to put a face on all the hard work you do.
